packages/examples/readme.md

```diff
@@ -1 +1,58 @@
-#TODO
+#Examples
+
+## SMS (via Twilio)
+
+`fixie-twilio-sms` provides a working example for interacting with a Fixie agent over SMS using [Twilio](https://twilio.com). To run the sample you will need to have a Twilio account and set-up a phone number to post to a webhook where this code is running. We recommend you use a service like [ngrok](https://ngrok.com) for testing.
+
+Additionally, you need to create a new file named `.env` and populate it with the following variables:
+
+```bash
+TWILIO_ACCOUNT_SID=<THIS_COMES_FROM_TWILIO>
+TWILIO_AUTH_TOKEN=<THIS_COMES_FROM_TWILIO>
+TWILIO_FROM_NUMBER=<YOUR_TWILIO_PHONE_NUMBER>
+WEBHOOK_URL=<NGROK_PUBLIC_URL>
+INCOMING_ROUTE="/messages"
+EXPRESS_PORT=3000
+FIXIE_AGENT_ID=<ID_OF_YOUR_FIXIE_AGENT>
+FIXIE_API_KEY=<YOUR_FIXIE_API_KEY>
+```
+
+If you are using ngrok, the `WEBHOOK_URL` should be the public URL that ngrok gives you (e.g. `https://22ab-123-45-67-89.ngrok-free.app`). This can also be the publicly accessible URL of your server if you have deployed the code publicly and not just on localhost.
+
+We used `/messages` for our incoming route. You can change this to something else just make sure to set the webhook in Twilio accordingly.
+
+`EXPRESS_PORT` can be changed as needed. `FIXIE_AGENT_ID` is the ID of the agent you want to talk to via SMS.
+
+With the above, you would have your Twilio number send webhooks to `https://22ab-123-45-67-89.ngrok-free.app/messages`.
+
+### Setting the Agent Prompt for SMS
+
+The mobile carriers have strict guidelines for SMS. We had to tune the prompt of our agent to be compatible with SMS (i.e. keep the messages short). Here is the prompt we created to enable the Fixie agent to be accessible via SMS.
+
+```bash
+You are a helpful Fixie agent. You can answer questions about Fixie services and offerings.
+
+The user is talking to you via text messaging (AKA SMS) on their phone. Your responses need to be kept brief, as the user will be reading them on their phone. Keep your response to a maximum of two sentences.
+
+DO NOT use emoji. DO NOT send any links that are not to the Fixie website.
+
+Follow every direction here when crafting your response:
+
+1. Use natural, conversational language that are clear and easy to follow (short sentences, simple words).
+1a. Be concise and relevant: Most of your responses should be a sentence or two, unless you're asked to go deeper. Don't monopolize the conversation.
+1b. Use discourse markers to ease comprehension.
```

Comment:
generally think discourse markers only make sense in a spoken context, for SMS they just take up space.